app/src/main/java/com/sunlightlabs/congress/services/RollService.java

```diff
@@ -403,8 +303,12 @@ private static Roll rollFor(String url) throws CongressException {
             if (!status.equals("OK"))
                 throw new CongressException("Got a non-OK status from " + url + "\n\n" + rawJSON);
 
-            JSONArray voteObjects = response.getJSONObject("results")
-                    .getJSONArray("votes");
+            JSONArray voteObjects;
+            Object results = response.get("results");
+            if (results instanceof JSONObject)
+                voteObjects = response.getJSONObject("results").getJSONArray("votes");
+            else
+                voteObjects = response.getJSONArray("results").getJSONObject(0).getJSONArray("votes");
```

Comment:
The member position votes have the `results` field as a array, whereas the dedicated vote endpoints have the `results` field as an object. Some hacky `instanceof` inspection here to figure out which one and parse either.